app-typescript/components/DateTimePicker.tsx

```diff
@@ -68,9 +68,9 @@ export class DateTimePicker extends React.PureComponent<IProps> {
                         preview={this.props.preview}
                         required={this.props.required}
                         hideClearButton={true}
-                        value={convertedValue}
+                        value={convertedValue ?? null}
```

Comment:
if `this.props.value` is `Date | null` it doesn't make sense to have `convertedValue` variable does it?